src/components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export const Calculator = ()=>{
+    return <div className="container">
+    <div className="output">
+      <span>0</span>
+    </div>
+    <button type="submit">AC</button>
+    <button type="submit">+/-</button>
+    <button type="submit">%</button>
+    <button type="submit" className="operand">
+      ÷
+    </button>
+    <button type="submit">7</button>
+    <button type="submit">8</button>
+    <button type="submit">9</button>
+    <button type="submit" className="operand">
+      x
+    </button>
+    <button type="submit">4</button>
+    <button type="submit">5</button>
+    <button type="submit">64</button>
```

Comment:
kindly fix this typo error because it doesn't follow the template design and would cause errors while working in the future 👍🏾 
![image](https://user-images.githubusercontent.com/60612329/229249155-29e159f1-951b-44e5-a8da-721a2f4797c3.png)